src/vcpkg/base/git.cpp

```diff
@@ -209,12 +209,12 @@ namespace vcpkg
 
     bool git_add_with_index(DiagnosticContext& context, const Path& git_exe, const Path& index_file, const Path& target)
     {
-        Command cmd(git_exe);
-        cmd.string_arg("-c").string_arg("core.autocrlf=false").string_arg("-c").string_arg("core.untrackedCache=false");
         RedirectedProcessLaunchSettings launch_settings;
```

Comment:
why remove `core.untrackedCache=false`?

Reply:
Because it didn't seem to actually fix the problem and it makes it slower